GPSTest/src/main/res/values/strings.xml

```diff
@@ -63,7 +63,10 @@
 
     <string name="gps_latitude_label">Lat:</string>
     <string name="gps_latitude_value">%1$.7f\u00B0</string>
-    <string name="gps_lat_lon_dms_value">%1$d\u00B0 %2$d\' %3$d\"</string>
+    <string name="gps_lat_dms_value">%1$s\t\u2007%2$02d\u00B0 %3$2d\' %4$05.2f\"</string>
```

Comment:
Let's move these formatting strings to `do_not_translate.xml` as well.  It doesn't look like there were any changes in the other translations, so they can all use the same format.